index.html

```diff
@@ -233,46 +250,57 @@ <h1 class="s">Skills</h1>
       </div>
     </ul>
     <!-- Footer Form Section -->
-
-    <section class="content-4 center">
-      <h1 class="clr-blue">
-        I'm always interested in hearing about new projects, so if you'd like to
-        chat please get in touch.
-      </h1>
-    </section>
-    <section class="center">
-      <form action="https://formspree.io/f/xeqwrjjn" method="POST">
-        <input
-          type="text"
-          class="in"
-          maxlength="30"
-          name="name"
-          placeholder="&nbsp;&nbsp;&nbsp;Full name"
-          required
-        />
-        <input
-          type="email"
-          class="in"
-          name="email"
-          placeholder="&nbsp;&nbsp;&nbsp;Email address"
-          required
-        />
-        <textarea
-          name="text"
-          id=""
-          cols="30"
-          rows="10"
-          maxlength="500"
-          required
-          placeholder="
+    <footer>
+      <div class="foot-flex">
+        <section class="content-4 center">
+          <h1 class="clr-blue">
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h1>
+        </section>
+        <section class="center">
+          <form action="https://formspree.io/f/xeqwrjjn" method="POST">
+            <input
+              type="text"
+              class="in"
+              maxlength="30"
+              name="name"
+              placeholder="&nbsp;&nbsp;&nbsp;Full name"
+              required
+            />
+            <br />
+            <input
+              type="email"
+              class="in"
+              name="email"
+              placeholder="&nbsp;&nbsp;&nbsp;Email address"
+              required
+            />
+            <br />
+            <textarea
+              name="text"
+              id=""
+              class="text-in"
+              cols="30"
+              rows="10"
+              maxlength="500"
+              required
+              placeholder="
         Hey I’d like to connect to..."
-        ></textarea>
-        <img class="foot" src="imgs/footer.png" alt="illustration" />
-        <br />
-        <button class="btn btn-2 btn-3" type="submit">Get in touch</button>
-        <p class="clr-blue">blesslinjerish@gmail.com</p>
+            ></textarea>
+            <img class="foot" src="imgs/footer.png" alt="illustration" />
+            <br />
+            <button class="btn btn-2 btn-3" type="submit">Get in touch</button>
```

Comment:
- [ ] Please make sure you follow the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-468&t=dWWSbacCPQ0hVj82-0)🎨 for styling the button here. This ensures you follow a given design to perfection which is a quality trait for a developer like yourself.
      ![firefox_kz6PEcoelc](https://user-images.githubusercontent.com/104968236/228802829-d7b93b1c-a0d0-4458-8a22-86209b80a4f2.png)
    - You're only missing the `:hover` state please add it.

Reply:
Hi @AbelG101 whats happening on the hover state I am unable to see the difference between enabled and hover . can you help ?